main/src/main/scala/org/clulab/polarity/ml/DeepLearningPolarityClassifier.scala

```diff
@@ -61,11 +62,16 @@ class DeepLearningPolarityClassifier() extends PolarityClassifier{
     HIDDEN_SIZE = config.getInt(configPath+".HIDDEN_SIZE")
     MLP_HIDDEN_SIZE = config.getInt(configPath+".MLP_HIDDEN_SIZE")
     N_EPOCH = config.getInt(configPath+".N_EPOCH")
+
+    dynetBlockMem = scala.math.min(config.getInt(configPath+".dynetMem")/4, 768).toString // set dynet block memory, max block memory is set to 768 for now.
```

Comment:
This still doesn't seem to be right. dynetMem is set to 512MB in the config file. Either increase it to 2048, or remove the division by 4 here.

Reply:
I want to double check, the dynetMem is set to 512 Mb, and each dynet memory block gets 128 Mb. But since there are 4 blocks, the total amount of memory allocated to dynet is still 512 Mb, which is the previous default setting. May I know if I need to increase the total dynet memory to 2048 Mb (which is 4 times of the previous total dynet memory)? 

Code to initialize dynet memory: 

initializeDyNet(mem = dynetBlockMem+","+dynetBlockMem+","+dynetBlockMem+","+dynetBlockMem)

Terminal output when running:  

[dynet] allocating memory: 128,128,128,128MB
[dynet] memory allocation done.

Reply:
Yes, that's what I'm saying. You should increase it to 2048 Mb in the config.

Reply:
Changed to 2048 Mb.